### PR TITLE
Minor fixes after testing

### PR DESCRIPTION
**Pull request by danishjamal104**

### CODE REVIEW

**Comment by danishjamal104** [comment]
@SubhamPramanik and @avikganguly01 please review and merge. Thanks :)

File: src/main/java/org/mifos/connector/ams/pesacore/zeebe/ZeebeWorkers.java [hunk 1]
```diff
@@ -57,8 +57,7 @@ public void setupWorkers() {
                         // Do stuff here
                         variables = job.getVariablesAsMap();
 
-                        JSONObject channelRequest = objectMapper.readValue(
-                                (String) variables.get("channelRequest"), JSONObject.class);
+                        JSONObject channelRequest = new JSONObject((String) variables.get("channelRequest"));
```
Comment (SubhamPramanik) [inline src/main/java/org/mifos/connector/ams/pesacore/zeebe/ZeebeWorkers.java:60]: Were you getting the error we discussed on the call because of this?

Comment (danishjamal104) [reply src/main/java/org/mifos/connector/ams/pesacore/zeebe/ZeebeWorkers.java:60]: that was in mpesa.

Comment (danishjamal104) [reply src/main/java/org/mifos/connector/ams/pesacore/zeebe/ZeebeWorkers.java:60]: The code reaches till [this](https://github.com/danishjamal104/ph-ee-connector-mpesa/blob/2c8fe7bdc76ee9273da37880a3266afb76bfd780/src/main/java/org/mifos/connector/mpesa/flowcomponents/transaction/CollectionResponseProcessor.java#L72) point, and zeebe works fine. But microservice throws error.

Comment (danishjamal104) [reply src/main/java/org/mifos/connector/ams/pesacore/zeebe/ZeebeWorkers.java:60]:
```
org.apache.camel.NoTypeConversionAvailableException: No type converter available to convert from type: org.apache.camel.util.json.JsonObject to the required type: java.nio.ByteBuffer with value {Body={stkCallback={MerchantRequestID=20424-27094491-1, CheckoutRequestID=ws_CO_02022022193315479351, ResultCode=0, ResultDesc=The service request is processed successfully., CallbackMetadata={Item=[{Name=Amount, Value=1}, {Name=MpesaReceiptNumber, Value=QB29JG2BQZ}, {Name=Balance}, {Name=TransactionDate, Value=20220202193338}, {Name=PhoneNumber, Value=254712087927}]}}}}
	at org.apache.camel.impl.converter.CoreTypeConverterRegistry.mandatoryConvertTo(CoreTypeConverterRegistry.java:275) ~[camel-base-3.12.0.jar!/:3.12.0]
	at org.apache.camel.impl.converter.CoreTypeConverterRegistry.mandatoryConvertTo(CoreTypeConverterRegistry.java:207) ~[camel-base-3.12.0.jar!/:3.12.0]
	at org.apache.camel.component.undertow.UndertowConsumer.sendResponse(UndertowConsumer.java:268) ~[camel-undertow-3.12.0.jar!/:3.12.0]
	at org.apache.camel.component.undertow.UndertowConsumer.handleRequest(UndertowConsumer.java:235) ~[camel-undertow-3.12.0.jar!/:3.12.0]
	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:387) ~[undertow-core-2.2.10.Final.jar!/:2.2.10.Final]
	at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:852) ~[undertow-core-2.2.10.Final.jar!/:2.2.10.Final]
	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35) ~[jboss-threads-3.1.0.Final.jar!/:3.1.0.Final]
	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2019) ~[jboss-threads-3.1.0.Final.jar!/:3.1.0.Final]
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1558) ~[jboss-threads-3.1.0.Final.jar!/:3.1.0.Final]
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1449) ~[jboss-threads-3.1.0.Final.jar!/:3.1.0.Final]
	at org.xnio.XnioWorker$WorkerThreadFactory$1$1.run(XnioWorker.java:1280) ~[xnio-api-3.8.4.Final.jar!/:3.8.4.Final]
	at java.base/java.lang.Thread.run(Thread.java:830) ~[na:na]
```

**Pull request merged**